docs/cow-protocol/concepts/benefits/additional-benefits.md

```diff
@@ -4,10 +4,23 @@ sidebar_position: 4
 
 # Additional benefits
 
-CoW Protocol's [intent-based trading model](../introduction/intents) provides a number of benefits, including support for expressing any intent, MEV protection, and price improvement. There are also a handful of additional benefits unique to CoW Protocol thanks to its specific architecture: 
+CoW Protocol's [intent-based trading model](https://docs.cow.fi/cow-protocol/concepts/introduction/intents) and [batch auctions mechanism](https://docs.cow.fi/cow-protocol/concepts/introduction/batch-auctions) come together to provide a number of benefits, including support for expressing any intent, MEV protection, and price improvement. 
```

Comment:
Please do not use absolute links for this as this is fragile (if moving domain etc, this will break in the new domain, etc).

Reply:
You should be able to just chop off the "https://docs.cow.fi" and use from the base href, ie. `/cow-proocol/concepts/introduction/intents`.

Reply:
Ok I just did this but when I click on them in preview I get a github error. This is originally why I used absolute links also, wondering if it works on your end.